packages/design-system/src/styles/components/_Alert.scss

```diff
@@ -76,6 +76,9 @@ $alert-icon-size: $spacer-5;
       margin-top: 0;
     }
   }
+  &.ds-c-alert--lightweight {
```

Comment:
[question] Does this class need extra specificity? Is that why the selector needs to be `.ds-c-alert.ds-c-alert--lightweight`?
If not, consider moving the `ds-c-alert--lightweight` selector to below/outside the `.ds-c-alert` block scope. It removes one degree of specificity and allows easier overwriting down stream

Reply:
Nevermind. I think I found my answer below AND why the lightweight needed extra specificity 🙃